src/sidebar/components/Tutorial.tsx

```diff
@@ -26,6 +26,7 @@ function LabeledIcon({ commandName, icon: Icon }: LabeledIconProps) {
           'mx-1 -mt-1', // Give horizontal space; pull up top margin a little
           'text-color-text-light inline',
         )}
+        aria-hidden
```

Comment:
Perhaps worth adding a note that we assume the icon is just a visual representation of the command and provides no extra information.